src/components/authentication/auth-config.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
```

Comment:
You have two of these files, one is in src/authentication as well

Reply:
removed